docker-compose.yml

```diff
@@ -34,16 +33,17 @@ services:
       RUST_LOG: debug
       RUST_BACKTRACE: 1
     volumes:
-      - /mnt/radicle/heartwood:/root/
+      - ${RAD_ROOT:-/mnt/radicle/heartwood}:/root/
```

Comment:
Just to make this more convenient for a "laptop" use, I wouldn't mind if we picked a sensible default, e.g. `${RAD_ROOT:-$HOME/.radicle/}` (as an aside: I'd be in favor or `~/.local/state/radicle` to conform with XDG specs).  We could flip it such that I'd override the default on the server via `.env.seed` file.  One fewer environment variable to configure that way.

Reply:
I am resolving this, as I think we've agreed on this in the main discussion of the PR.